core/src/main/scala/io/finch/syntax/ToTwitterFuture.scala

```diff
@@ -0,0 +1,17 @@
+package io.finch.syntax
+
+import com.twitter.util.Future
+
+trait ToTwitterFuture[F[_]] {
+
+  def apply[A](f: F[A]): Future[A]
+
+}
+
+object ToTwitterFuture {
+
+  implicit val identity: ToTwitterFuture[Future] = new ToTwitterFuture[Future] {
+    override def apply[A](f: Future[A]): Future[A] = f
```

Comment:
No need for `override`.

Reply:
Thanks for comments!

I was playing around with testing of this beast, but went way too far with scalacheck generators. Going to try again